class.jetpack.php

```diff
@@ -6932,22 +6933,23 @@ public static function build_raw_urls( $url ) {
 	 */
 	public static function dns_prefetch( $new_urls = null ) {
 		static $prefetch_urls = array();
+		static $relation_type = 'dns-prefetch'; /* Should this be a passed param, to allow different link types*/
```

Comment:
Yes, it would be best.

Reply:
I've just realised, whilst it makes sense to pass this as a parameter for now, if you were to look at using other rel types in the future, you'd probably want to rename the function. 
In theory as well, in it's current state the function could probably be cleaned up to wrap the entire entire thing in the `if ($relation_type == 'dns-prefetch)` rather than having two nested. That way, you only have to maintain the single `if` and adding others would be _fairly_ easy from then on - again with a single `if`.
Hopefully I'm making sense there!